### PR TITLE
added apply here button

### PR DESCRIPTION
**Pull request by WelehoBRUDER**

### CODE REVIEW

File: frontend/app/src/pages/Connected/Home/index.js [hunk 1]
```diff
@@ -1,266 +1,179 @@
-import React from 'react'
-import './style.scss'
+import React from "react";
+import "./style.scss";
```
Comment (Earlo) [inline frontend/app/src/pages/Connected/Home/index.js:2]: Seems your linter is configured differently than it should.

Comment (Earlo) [reply frontend/app/src/pages/Connected/Home/index.js:2]: Do you have prettier installed?

.prettierrc on root of the project should be

```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true
}

```

**Review by Earlo — CHANGES_REQUESTED** [review]
Install prettier. 

Other than that looks good.

File: frontend/app/src/pages/Connected/Home/index.js (hunk 1 above)
Comment (1UPNuke) [reply frontend/app/src/pages/Connected/Home/index.js]: Fixed

**Review by Earlo — APPROVED** [review]
Looks good : )

**Pull request merged**